SCloudRandomizer/MainViewController.m

```diff
@@ -60,19 +63,17 @@ - (void)viewDidLoad {
     [self.lblTitleValue setText:@""];
     self.currentSongNumber = 3;
```

Comment:
I thought I had cleaned this up in an earlier change? Do we need `self.currentSongNumber = 3` ?  Also perhaps we should do the UI updates like clearing the labels and drawing borders in a separate methods to make it cleaner?